include/abb_librws/rws_interface.h

```diff
@@ -556,6 +556,19 @@ class RWSInterface
                                   const std::string& tool = "",
                                   const std::string& wobj = "");
 
+  /**
+   * \brief A method for retrieving the data of a RAPID symbol in raw text format.
+   *
+   * See the corresponding "setRAPIDSymbolData(...)" method for examples of RAPID symbols in raw text format.
+   *
+   * \param task for the name of the RAPID task containing the RAPID symbol.
+   * \param module for the name of the RAPID module containing the RAPID symbol.
+   * \param name for the name of the RAPID symbol.
+   *
+   * \return std::string containing the data. Empty if not found.
```

Comment:
I guess "empty string" is not a RAPID data-type, but would this method be a candidate for an exception?

Reply:
Yes, it is not a RAPID data type.

And yes, it would be a candidate for throwing an exception, though I would like to do it in another PR. Mainly because  there are some other similar methods that could also benefit from it as well, and I would like to take a look at them at the same time.